.go_version

```diff
@@ -0,0 +1 @@
+1.13.0
```

Comment:
WDYT about creating a "default" location for a file similar to what `nvm use` does. to get rid of the cat magic?

Reply:
I don't have an issue with such a feature as [described for .nvmrc](https://github.com/nvm-sh/nvm/blob/master/README.md#nvmrc). But I also don't see a whole lot of extra value in either. It might help establish a convention for specifying the Go version in a project.

Reply:
I would expect it helps for cross stack compatibility. Will go for now with the shell version. Worth opening an issue in gvm?

Reply:
Definitely open an issue for it. 👍

Reply:
https://github.com/andrewkroh/gvm/issues/20